site/content/user-guide/device-registry.md

```diff
@@ -3,17 +3,38 @@ title = "Device Registry"
 weight = 205
 +++
 
+The Device Registry component is an example implementation of the following Hono APIs:
+
+* [Tenant API]({{< relref "api/Tenant-API.md" >}})
+* [Device Registration API]({{< relref "api/Device-Registration-API.md" >}})
+* [Credentials API]({{< relref "api/Credentials-API.md" >}})
+
+It implements AMQP 1.0 API endpoints and provides JSON based file persistence for data. 
+
 In addition to the AMQP 1.0 based API endpoints, the Device Registry also exposes HTTP resources for managing the contents of the registry.
 
-<!--more-->
```

Comment:
please do not remove this separator, it makes sure that the overview page only contains the text above